src/compiler/moduleSpecifiers.ts

```diff
@@ -513,7 +513,7 @@ function computeModuleSpecifiers(
             info,
             compilerOptions,
             host,
-            options.overrideImportMode || importingSourceFile.impliedNodeFormat,
+            options.overrideImportMode,

```

Comment:
This was a stray debugging change (the result of which was, removing this causes no test failures 🫤), revertig